api/app/routers/fba_calc.py

```diff
@@ -26,6 +27,7 @@
 def prepare_response(
         requested_station: StationRequest,
         wfwx_station: WFWXWeatherStation,
+        fba_station: FBACalculatorWeatherStation,
```

Comment:
It's so confusing tracing where values are coming from. E.g. the FFMC/ISI could be either from the raw_daily, or calculated by cffdrs using the wind speed. I don't like this at all - but don't have a good idea right.

Reply:
I guess I'm saying I'd prefer we not have fba_station: FBACalculatorWeatherStation here - but I don't have a better answers. It's just all so confusing having the input to the calculation also be the input to the thing that formats the response.